cmd/skopeo/utils.go

```diff
@@ -28,6 +28,9 @@ func contextFromGlobalOptions(c *cli.Context, flagPrefix string) (*types.SystemC
 			return nil, err
 		}
 	}
+	if c.BoolT(flagPrefix + "no-creds") {
```

Comment:
(This affects almost all commands; so the documentation, and bash completions, should also be updated everywhere.)

Reply:
👍

Reply:
`BoolT` means “default to true”; this should default to false.